interface/patient_file/summary/demographics.php

```diff
@@ -1034,13 +1034,11 @@ function setMyPatient() {
                     $pl = (AclMain::aclCheckIssue('medical_problem')) ? 1 : 0;
                     $meds = (AclMain::aclCheckIssue('medication')) ? 1 : 0;
                     $rx = (!$GLOBALS['disable_prescriptions'] && AclMain::aclCheckCore('patients', 'rx')) ? 1 : 0;
-                    $cards = $allergy + $pl + $meds + $rx;
+                    $cards = $allergy + $pl + $meds;
                     $col = "p-1 ";
 
-                    if ($cards > 0) {
```

Comment:
Guessing removing above comparison will  break for folks that do not have acl access to any of the above 3 cards (ie. receptionist) since I recently just added the above comparison to fix that bug :)

Reply:
btw, the break was a dividing by zero error, which is why ensured $cards was  > 0

Reply:
also, don't worry about fixing/testing this minor issue. plan to test and fix it, if needed, tonight :)